src/pages/ProductDetail/ProductDetailMain/ProductDetailMain.js

```diff
@@ -0,0 +1,540 @@
+    tabMenu.current.style.fontWeight = "700";
+    tabMenu.current.style.borderBottom = "2px solid black";
+    tabMenu.current.style.color = "black";
+  };
+
+  const handleReviewClick = () => {
+    setReviewClick(!reviewClick);
+  };
+
+  // const handleDeleteClick = () => {
+  //   fetch("http://10.58.7.33:8000/users/review", {
+  //     method: "DELETE",
+  //     body: JSON.stringify({
+  //       user_id: "9",
+  //       product_id: "1",
+  //       content: reviewValue,
+  //       rating: Number(ratingValue),
+  //       image_url: fileImg,
+  //     }),
+  //   });
+  // };
+  const handleWriteBtn = () => {
+    setOpenModal(true);
+  };
+
+  const handleRatingInput = e => {
+    setRatingValue(e.target.value);
+  };
+
+  const handleReviewInput = e => {
+    setReviewValue(e.target.value);
+  };
+
+  const handleReviewPost = () => {
+    setIsPosted(!isPosted);
+    setOpenModal(false);
+    setReviewValue("");
+    setRatingValue(0);
+    fetch("http://10.58.7.33:8000/users/review", {
+      method: "POST",
+      body: JSON.stringify({
+        user_id: "9",
+        product_id: "1",
+        content: reviewValue,
+        rating: Number(ratingValue),
+        image_url: fileImg,
+      }),
+    });
+    // .then(response => response.json())
+    // .then(result => setDetail(result));
+  };
+  console.log(detail);
+
+  const handleImgInput = e => {
+    setFileImg(URL.createObjectURL(e.target.files[0]));
+  };
+
+  useEffect(() => {
+    fetch("http://10.58.7.33:8000/products/1/1", {
+      method: "GET",
+    })
+      .then(res => res.json())
+      .then(res => setDetail(res.product));
+  }, [isPosted]);
+
+  useEffect(() => {
+    reviewClick
+      ? (reviewBox.current.style.overflow = "visible") &&
+        (reviewBox.current.style.height = "auto")
+      : (reviewBox.current.style.overflow = "hidden") &&
+        (reviewBox.current.style.height = "220px");
+  }, [reviewClick]);
+
+  const ratingArr = [];
+  const initialValue = 0;
+
+  detail[0]?.reviews.map(review => {
+    return ratingArr.push(review.rating);
+  });
+
+  const ratingAmount = ratingArr.reduce(
+    (prev, curr) => prev + curr,
+    initialValue
+  );
+
+  const starAverage = ratingAmount / detail[0]?.reviews.length;
+  return (
+    <LeftSide>
+      <ThumbBox>
+        <ThumbImg src={detail[0]?.thumb_img} />
+        <ThumbCoupon>{String(coupon)[0]}만원 쿠폰</ThumbCoupon>
+      </ThumbBox>
+      <TabMenus>
+        {TAB_MENU.map(menu => {
+          return (
+            <Link key={menu.id} to={menu.id} spy={true} smooth={true}>
+              <TabMenu onClick={handleTabClick} ref={tabMenu}>
+                {/* <TabMenu onClick={handleTabClick}  ref={tabMenu} color fontWeight border> */}
+                {menu.title}
+              </TabMenu>
+            </Link>
+          );
+        })}
+      </TabMenus>
+
+      <Banner bgColor="#77B053">
+        세대별 TOP 50 클래스 공개!
+        <br />
+        <Text color="white" margin="8px 0 0 0" fontSize>
```

Comment:
* 콘솔 탭 열어보면, warning이 있을꺼에요
* 스타일링을 조건에 `fontSize`가 있으면 8px 없으면 14px 로 되있는데, 
* 조건으로 fontSize보단, 적용 하려는 UI 라면~ 00px, 아니라면 ~00px 로 조건 명이 바뀌면 더 좋을 것같고,
* fontSize는 이미 존재하는 속성명 이어서, 만약 존재하는 속성명을 써야 한다면prefix로 `$`을 속성명에 붙여주면 된다는 기능이 styled-component 5.1v에 추가됐습니다.
```suggestion
        <Text color="white" margin="8px 0 0 0" $fontSize>
```